src/Data/ByteString/Base16/Internal/W64/Loop.hs

```diff
@@ -236,3 +155,15 @@ decodeLoop !dfp !hi !lo !dptr !sptr !end = go dptr sptr 0
             poke @Word32 dst zz
             go (plusPtr dst 4) (plusPtr src 8) (n + 4)
 {-# INLINE decodeLoop #-}
+
+lenientLoop
+  :: ForeignPtr Word8
+  -> Ptr Word8
+  -> Ptr Word8
+  -> Ptr Word8
+  -> Ptr Word8
+  -> Ptr Word8
+  -> Int
+  -> IO ByteString
+lenientLoop !dfp !hi !lo !dptr !sptr !end !nn =
+  W32.lenientLoop dfp hi lo dptr sptr end nn
```

Comment:
Turns out leniency only really makes sense for two values at a time for hex, so just reusing `W16.lenientLoop` is fine.